packages/core/src/codewhisperer/service/securityScanHandler.ts

```diff
@@ -82,18 +82,20 @@ export async function listScanResults(
             // Do not use .. in between because there could be multiple project paths in the same parent dir.
             const filePath = path.join(projectPath, key.split('/').slice(1).join('/'))
             if (existsSync(filePath) && statSync(filePath).isFile()) {
+                const document = await vscode.workspace.openTextDocument(filePath)
                 const aggregatedCodeScanIssue: AggregatedCodeScanIssue = {
                     filePath: filePath,
-                    issues: issues.map((issue) => mapRawToCodeScanIssue(issue, editor, jobId, scope)),
+                    issues: issues.map((issue) => mapRawToCodeScanIssue(issue, jobId, scope, document)),
```

Comment:
nit: you can make this change even smaller by keeping the position of the editor/document argument the same